tests/topotests/lib/common_config.py

```diff
@@ -3435,7 +3447,20 @@ def socat_send_pim6_traffic(
             )
 
         rnode.run("chmod 755 {}".format(traffic_shell_script))
-        output = rnode.run("{} &> /dev/null".format(traffic_shell_script))
+        output = rnode.run("{} &>/dev/null & echo $!".format(traffic_shell_script))
```

Comment:
Just to double check: `& echo` or `&& echo`?

Reply:
Hi @ton31337 , I used `& echo $!`  to get process id on console, I am reading it and saving in file to use it later.